Question13.java

```diff
@@ -0,0 +1,32 @@
+class Solution {
+    public int romanToInt(String s) {
+        HashMap<String, Integer> map = new HashMap<String,Integer>();
+        map.put("I", 1);
+        map.put("V", 5);
+        map.put("X", 10);
+        map.put("L", 50);
+        map.put("C", 100);
+        map.put("D", 500);
+        map.put("M", 1000);
+        System.out.println(s);
```

Comment:
Once a debugging code served its purpose, you should delete or at least comment it out.